Doc/library/os.path.rst

```diff
@@ -488,6 +488,38 @@ the :mod:`glob` module.)
       Accepts a :term:`path-like object`.
 
 
+.. function:: splitroot(path)
+
+   Split the pathname *path* into a triad ``(drive, root, tail)`` where:
+
+   1. *drive* is a mount point or the empty string;
```

Comment:
```suggestion
   1. *drive* is a device name, mount point or the empty string;
```

The `spltdrive()` documentation says that a drive is a mount point, but that's not generally true. A mount point is a path that's mounted by a filesystem directory. A UNC share is a mount point for a redirected or virtual filesystem, but a drive such as "D:" is usually the name of a volume device. The automatic mount point is the root path "D:\\", assuming the volume is even formatted.

Mapped drives and substitute drives are logical device names, though they can behave more like symlinks. For an example of the latter, if substitute drive "S:" targets "\\??\\C:\\Windows\\System32", then a relative symlink "S:\\link.exe" -> "..\\notepad.exe" resolves to "C:\\Windows\\notepad.exe". If "S:" were a real device name, then trying to traverse "S:\\link.exe" would fail as invalid.

As a pure path operation, we don't check the device type or whether it exists. Any device name is accepted as a drive, including known serial devices such as "\\\\.\\NUL", "\\\\.\\CON" and "\\\\.\\COM1".